independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/CodestartProjectGeneration.java

```diff
@@ -34,8 +33,6 @@ static void generateProject(final CodestartProjectDefinition projectDefinition,
 
         log.debug("processed shared-data: %s" + data);
 
-        final Codestart projectCodestart = projectDefinition.getRequiredCodestart(CodestartType.PROJECT);
```

Comment:
@glefloch why removing that line?

Reply:
The variable is never used. And as this is a get, I think doesn't do anything. Right?

Reply:
It checks that the projectDefinition contains a codestart of type `CodestartType.PROJECT`, you can remove the variable though :)

Reply:
oh ok, I will reintroduce that line without the variable then :)

Reply:
done ;)